src/components/Tool/FixPhoneNumbers/FixPhoneNumbers.tsx

```diff
@@ -232,100 +196,100 @@ const FixPhoneNumbers: React.FC = () => {
-                      className={classes.nativeSelect}
-                      value={defaultSource}
-                      onChange={(event: React.ChangeEvent<HTMLSelectElement>) =>
-                        handleSourceChange(event)
-                      }
-                    >
-                      <option value="MPDX">MPDX</option>
-                      <option value="DataServer">DataServer</option>
-                    </NativeSelect>
-                    <Button className={classes.buttonBlue}>
-                      <Icon
-                        path={mdiCheckboxMarkedCircle}
-                        size={0.8}
-                        className={classes.buttonIcon}
+                      <NativeSelect
+                        input={<StyledInput />}
+                        className={classes.nativeSelect}
+                        value={defaultSource}
+                        data-testid="source-select"
+                        onChange={(
+                          event: React.ChangeEvent<HTMLSelectElement>,
+                        ) => handleSourceChange(event)}
+                      >
+                        <option value="MPDX" data-testid="source-option-mpdx">
+                          MPDX
+                        </option>
+                        <option
+                          value="DataServer"
+                          data-testid="source-option-dataserver"
+                        >
+                          DataServer
+                        </option>
+                      </NativeSelect>
+                      <Button
+                        className={classes.buttonBlue}
+                        data-testid="source-button"
+                      >
+                        <Icon
+                          path={mdiCheckboxMarkedCircle}
+                          size={0.8}
+                          className={classes.buttonIcon}
+                        />
+                        {t('Confirm {{amount}} as {{source}}', {
+                          amount: dataState.length,
+                          source: defaultSource,
+                        })}
+                      </Button>
+                    </Box>
+                  </Box>
+                </Grid>
+                <Grid item xs={12}>
+                  {dataState.map((person, index) => (
+                    <Contact
+                      name={`${person.firstName} ${person.lastName}`}
+                      key={person.id}
+                      contactIndex={index}
+                      numbers={person.phoneNumbers.nodes}
+                      handleChange={handleChange}
+                      handleDelete={handleDeleteModalOpen}
+                      handleAdd={handleAdd}
+                      handleChangePrimary={handleChangePrimary}
+                    />
+                  ))}
+                </Grid>
+                <Grid item xs={12}>
+                  <Box className={classes.footer}>
```

Comment:
Tbh I never really used the styled API and initially Scotty said using makeStyles was fine (I saw in other components too), but if there is advantage to using style I can just use it moving forward

Reply:
I've really liked using the styled api. I've always used Emotion or styled-components with React, so it just feels natural for me personally. Feels more... "React-esque" I guess you could say haha? Not sure the specific benefits of either or, maybe more of just a preference? I know styled allows you to pass props to your styled component in order to dynamically change style values, but maybe ```makeStyles``` can do that too 🤷‍♂️ I know there was a lot of changes for ```makeStyles``` related to v5 MUI upgrade as well.

Reply:
Generally speaking, I think we will want to be consistent across the project as much as possible.

Reply:
I agree. I feel like we came to a consensus to move away from ```makeStyles``` and we slowly started converting components that were using it over to the styled api. @OzzieOrca are we still moving away from ```makeStyles```? If so, it doesn't necessarily have to be addressed in this pr, but probably should be in a future pr.

Reply:
Ya seems like we should be moving away from `makeStyles`. https://mui.com/guides/migration-v4/#1-use-styled-or-sx-api

I think I intended to say `makeStyles` was ok for a single PR since you'd already written it and was ok for existing code but that we prefered the `styled` API. I should have brought it up in other PRs but was focused on other things. Now that v5 seems to have dropped support for it, I think it's more important that we stop writing code that will need changes during a migration to v5.